core/src/com/ooutofmind/Reg.java

```diff
@@ -0,0 +1,29 @@
+package com.ooutofmind;
+
+import com.badlogic.gdx.utils.Disposable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Reg implements Disposable {
+    private static final List<Disposable> toBeRemoved = new ArrayList<>();
+
+    public static final Reg i = new Reg();
```

Comment:
instance?

Reply:
nope ;) welcome to fucking gamedev prototyping